include/ignition/math/detail/Capsule.hh

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_MATH_DETAIL_CAPSULE_HH_
+#define IGNITION_MATH_DETAIL_CAPSULE_HH_
+
+#include <ignition/math/Inertial.hh>
+
+namespace ignition
+{
+namespace math
+{
+
+//////////////////////////////////////////////////
+template<typename T>
```

Comment:
the other shape classes are using `T` in their `detail` headers. I'll leave it as `T` for now since it's so much easier to read; we can revisit if there's a more concrete example of the problem

Reply:
Sounds good.